pom.xml

```diff
@@ -1219,6 +1219,7 @@
     <activemqVersion>5.10.0</activemqVersion>
     <atomikosVersion>3.9.2</atomikosVersion>
     <camelVersion>2.14.1</camelVersion>
+    <elasticsearch-version>1.0.0_4</elasticsearch-version>
```

Comment:
Please update this to 'elasticsearchVersion' so that it matches all other properties also.